test/utils.test.js

```diff
@@ -157,4 +157,22 @@ describe('utils', () => {
     const result = truncateHexString(hex256, desiredBits);
     expect(result.length).to.equal(expectedLength);
   });
+
+  describe('isSameDay', () => {
+    it('Returns true if two dates are on the same day', () => {
```

Comment:
Not a huge deal but at least for our test suite we don't uppercase the first letter of the sentence